web/src/app/administration/components/classification-details/classification-details.component.ts

```diff
@@ -131,6 +134,11 @@ export class ClassificationDetailsComponent implements OnInit, OnDestroy {
     }
   }
 
+  onCopy() {
+    this.store.dispatch(new SetActiveAction(ACTION.COPY));
+    this.classification.key = '';
```

Comment:
shouldn't we make a copy first? Are we otherwise overriding the values from the classification we copied from? Can you verify this please?

Reply:
Because we use the store we always just work with a local copy, and when switching to the old classification we retrive the newest version of it, which resets unsaved changes. If saved the classification does not overwrite the old one